### PR TITLE
contour guide

### PR DESCRIPTION
**Pull request by paulczar**
Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

### CODE REVIEW

File: content/guides/kubernetes/contour-to-ingress-and-beyond.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
+---
+title:  "Getting Started with Contour - To Ingress and Beyond"
+linkTitle: "Getting Started with Contour - To Ingress and Beyond"
+parent: Contour to Ingress and Beyond
+weight: 2
+topics:
+- Kubernetes
+- Tanzu
```
Comment (bryanfriedman) [inline content/guides/kubernetes/contour-to-ingress-and-beyond.md:8]: Remove "Tanzu" as a topic and that should fix the broken link check. Topic has to be one of the existing topics.

File: content/guides/kubernetes/contour-to-ingress-and-beyond.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+- service mesh
+- network
+# Author(s)
+team:
+- Paul Czarkowski
+---
+
+### Introduction to Contour
+
+Need to write some details about Contour here
```
Comment (bryanfriedman) [inline content/guides/kubernetes/contour-to-ingress-and-beyond.md:25]: FYI, adding an article on Contour as part of this PR: https://github.com/vmware-tanzu/tanzu-dev-portal/pull/516/files - Might be good enough as a "what is" to precede this one and cover some of the intro piece here.

**Comment by bryanfriedman** [comment]
I know this is WIP but just added a few comments as FYIs...

File: content/guides/kubernetes/contour-to-ingress-and-beyond.md [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+
+### Installing Contour 1.12.0
+
+Since version 1.11.0 we've got two primary options for installing Contour, A singleton install from manifests, or by using the [Operator](https://projectcontour.io/resources/deprecation-policy/) (which is currently in Alpha). Since we only plan to install Contour once on the cluster, we can stick to the safer method of using the Contour provided manifests.
+
+You can install Contour directly from the manifests provided by the project, however best practice would have you download them locally first for validation and repeatability.
+
+1. Download contour installation manifests
+    ```bash
+    wget https://projectcontour.io/quickstart/contour.yaml
```
Comment (youngnick) [inline content/guides/kubernetes/contour-to-ingress-and-beyond.md:54]: Note that you can also do
```bash
wget https://projectcontour.io/quickstart/v1.12.0/contour.yaml
```
And get the `v1.12.0` version. That will work with any tagged release ( the base quickstart will get you the latest release).

**Review by youngnick — COMMENTED** [review]
This looks great @paulczar! Thanks from all the Contour team!

File: content/guides/kubernetes/contour-to-ingress-and-beyond.md (hunk 3 above)
Comment (paulczar) [reply content/guides/kubernetes/contour-to-ingress-and-beyond.md:54]: perfect! that was something I was going to figure out before publishing :)

File: content/guides/kubernetes/contour-to-ingress-and-beyond.md (hunk 2 above)
Comment (paulczar) [reply content/guides/kubernetes/contour-to-ingress-and-beyond.md:25]: yeah I figured we had a few examples that use contour, that we could skip the real basics

**Pull request merged**